plugin/pxr/maya/lib/usdMaya/testenv/UsdExportPointInstancerTest/InstancerTest.ma

```diff
@@ -0,0 +1,262 @@
+//Maya ASCII 2016 scene
```

Comment:
This file was actually removed by
https://github.com/Autodesk/maya-usd/commit/7cd3c48e173434ec9cdb074f7a2adfade4fd0258
Will fix.